api/v1/services/user.py

```diff
@@ -376,6 +386,17 @@ def unfollow_user(self, db: Session, user_id: str, user: User):
             )
 
         user.followings.remove(user_to_unfollow)
+
+        notification1 = Notification(
```

Comment:
Remove this notification. No need sending notification to the user who unfollowed another user.